client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -58,17 +60,18 @@ class OrderDetailsTable extends Component {
 	}
 
 	recalculateRefund = () => {
-		if ( ! this.props.order ) {
+		const { order } = this.props;
+		if ( ! order ) {
 			return 0;
 		}
 		const subtotal = sum( this.state.quantities.map( ( q, i ) => {
-			if ( ! this.props.order.line_items[ i ] ) {
+			if ( ! order.line_items[ i ] ) {
 				return 0;
 			}
-			const price = parseFloat( this.props.order.line_items[ i ].price );
-			const tax = parseFloat( this.props.order.line_items[ i ].total_tax ) / this.props.order.line_items[ i ].quantity;
+			const price = parseFloat( order.line_items[ i ].price );
+			const tax = getOrderLineItemTax( order, i ) / order.line_items[ i ].quantity;
 			const taxIncludedPrice = price + tax;
-			if ( this.props.order.prices_include_tax ) {
+			if ( order.prices_include_tax ) {
```

Comment:
This conditional would make more sense to me if it was before the taxIncludedPrice calculation, since neither the test nor its block use taxIncludedPrice. What do you think?